doc/api/magic-plot.md

```diff
@@ -151,6 +153,12 @@ When plotting a histogram, it divides a range with the number of bins - 1 to cal
 %sqlplot histogram --table penguins.csv --column body_mass_g  --bins 100
 ```
 
+### Specifying breaks
+
```

Comment:
Would breaks that cause an error be the ill-defined cases?

Reply:
yes, any conditions that leads you to raise an error is what I meant by ill-defined

Reply:
Some context added in magic-plot.md

Reply:
I agree. I think we can elaborate a bit more. In this [example](https://www.tutorialspoint.com/how-to-define-the-breaks-for-a-histogram-using-ggplot2-in-r), they show a nice example of what can be done with breaks.

also, I think we should add a note that we can't use both `--bins` and `--breaks` since it looks like ggplot2 allows it (why can't we use both of them?)

Reply:
Added extra information about what breaks are used for and that we can't use both `--bins` and `--breaks`.